ui/appui-react/src/appui-react/icons/SvgAssetClassificationHide.tsx

```diff
@@ -0,0 +1,36 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+/** @packageDocumentation
+ * @module Utilities
+ */
+
+import * as React from "react";
+
+/** @internal */
+export function SvgAssetClassificationHide() {
+  return (
+    <svg
+      version="1.1"
+      id="Layer_1"
+      xmlns="http://www.w3.org/2000/svg"
+      xmlnsXlink="http://www.w3.org/1999/xlink"
+      x="0px"
+      y="0px"
+      viewBox="0 0 16 16"
+      enableBackground="new 0 0 16 16;"
+      xmlSpace="preserve"
+    >
+      <path
+        d="M5,0l5,5l-5,5L0,5L5,0z M7.4131,15.1504l6.7118-6.7118l0.8496,0.8496L8.2627,16L7.4131,15.1504z M8.0099,14.1417
+	l0.6608-0.6608c-0.515-0.2944-0.9326-0.7329-1.2015-1.2616c0.8072-1.2845,2.2079-2.0752,3.7247-2.1027
+	c0.2419-0.0099,0.484,0.0103,0.7209,0.0601l0.781-0.781c-0.4911-0.1226-0.9957-0.1832-1.5019-0.1802
+	c-2.047-0.0085-3.9167,1.1601-4.8061,3.0038C6.7581,12.9864,7.316,13.6476,8.0099,14.1417z M14.438,10.2968l-0.6608,0.6608
+	c0.4832,0.3332,0.8923,0.7627,1.2015,1.2616c-0.8088,1.2828-2.2085,2.073-3.7247,2.1027c-0.2631-0.0194-0.5242-0.0595-0.781-0.1202
+	l-0.781,0.781c1.2744,0.3811,2.6451,0.2741,3.8449-0.3004c1.0691-0.5284,1.9347-1.394,2.4631-2.4631
+	C15.6321,11.4682,15.0979,10.8108,14.438,10.2968z"
+      />
+    </svg>
```

Comment:
```suggestion
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16"><path d="m5 0 5 5-5 5-5-5 5-5zm2.413 15.15 6.712-6.711.85.85L8.263 16l-.85-.85zm.597-1.008.66-.661a3.041 3.041 0 0 1-1.2-1.262 4.495 4.495 0 0 1 3.724-2.102c.242-.01.484.01.72.06l.782-.781a6.053 6.053 0 0 0-1.502-.18 5.312 5.312 0 0 0-4.806 3.003 5.06 5.06 0 0 0 1.622 1.923zm6.428-3.845-.66.66c.482.334.892.763 1.2 1.262a4.508 4.508 0 0 1-3.724 2.103 4.99 4.99 0 0 1-.781-.12l-.781.78a5.352 5.352 0 0 0 3.845-.3A5.433 5.433 0 0 0 16 12.22a5.509 5.509 0 0 0-1.562-1.922z"/></svg>
```

Ran through [SVGOMG](https://jakearchibald.github.io/svgomg/).

Reply:
Ran all SVGs through [SVGviewer](https://www.svgviewer.dev/)